kpops/components/base_components/pipeline_component.py

```diff
@@ -27,18 +27,16 @@ class PipelineComponent(BaseDefaultsComponent):
 
     :param name: Component name
     :type name: str
-    :param from_: Topic(s) and/or components from which the component will read
-        input, defaults to None
-    :type from_: FromSection, optional
-    :param app: Application-specific settings, defaults to None
-    :type app: object, optional
-    :param to: Topic(s) into which the component will write output,
-        defaults to None
-    :type to: ToSection, optional
     :param prefix: Pipeline prefix that will prefix every component name.
         If you wish to not have any prefix you can specify an empty string.,
         defaults to "${pipeline_name}-"
     :type prefix: str, optional
+    :param from: Topic(s) and/or components from which the component will read
```

Comment:
Why did you rename `from_` to `from` in the docstr?

Reply:
done by mistake, I thought the parameter can be used with the alias, but that's not the case. will revert